pages/instructorAssessmentManualGrading/assessment/assessment.sql

```diff
@@ -9,14 +20,11 @@ WITH instance_questions_with_submission AS (
         JSONB_AGG(DISTINCT jsonb_build_object('user_id', agu.user_id, 'name', agu.name, 'uid', agu.uid)) FILTER (WHERE iq.requires_manual_grading AND iq.assigned_grader IS NOT NULL) AS assigned_graders,
         JSONB_AGG(DISTINCT jsonb_build_object('user_id', lgu.user_id, 'name', lgu.name, 'uid', lgu.uid)) FILTER (WHERE iq.last_grader IS NOT NULL) AS actual_graders
     FROM
-        assessment_questions aq
+        selected_assessment_questions aq
```

Comment:
I'm probably missing something obvious, but this one is `selected_assessment_questions`, whereas the first CTE is now named `selected_instance_questions`? I'd also expect the `LEFT JOIN instance_questions_with_submission iqs` in the final `SELECT` at the end of this whole query to change.

Reply:
Ugh, thanks, this is what happens when I manually copy bits from my working test query. Fixed in 25986610f8734f72d4edf1c352a9693dbf6de75b

We don't need to change the final `LEFT JOIN instance_questions_with_submission iqs` because this new query still computes exactly the same `instance_questions_with_submission` CTE, it just does it more efficiently. Or am I misunderstanding what you mean?

Reply:
And *this* is what happens when I review PRs too early in the morning 😂 you're absolutely correct, ignore that second part.

Reply:
Hahaha, between the two of us we are almost managing to have one functioning brain.